packages/smart-wallet/src/smartWallet.js

```diff
@@ -423,7 +403,18 @@ const SmartWalletKit = defineVirtualFarClassKit(
               invitationPurse,
               offerToInvitationMakers.get,
             ),
-            purseForBrand: brandPurses.get,
+            // async so we consistently return a promise
+            purseForBrand: async brand => {
```

Comment:
alternately, in a way that doesn't need explaining:
```suggestion
            /** @returns {Promise<Brand>} */
            purseForBrand: brand => {
```
then the `return brandPurses.get(brand);` would have a squiggly and force,
```
return Promise.resolve(brandPurses.get(brand));
```

Reply:
That exposed a conflict with `withdrawGive` where the purse for a brand is expected to be available synchronously. Ugh.

Reply:
ah... it's ok to have promises in `keywordPaymentPromises`...